tests/functional/test_issue-228/gen_plan.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+set -eo pipefail
+
+docker run --rm -v $(pwd):/app -w /app hashicorp/terraform:light init
+docker run --rm -v $(pwd):/app -w /app hashicorp/terraform:light plan --out plan.out
+docker run --rm -v $(pwd):/app -w /app hashicorp/terraform:light show --json plan.out | jq > plan.out.json 
```

Comment:
Hmm why do we have this here ? We should only have plan.out.json and the feature file in the functional tests for the future testing.

I suppose you created this for your local workstation ?

Reply:
Yes, sorry. Checked in by mistake.

Reply:
Removed.